Babel issue in projects not using babel

I have never used Babel and this might be a very basic question.

I have a typescript project that doesn't use babel or webpack and I am using apollo-server. When I tried to use `graphql-query-complexity` I got the following error

```
Error: Cannot find module 'babel-runtime/helpers/typeof'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\garp\Source\Data Platform\teams-adp\node_modules\graphql-query-complexity\dist\QueryComplexity.js:7:16)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
```

I found that you are using babel in devDependencies. So I tried adding those to my project's devDependencies and the issue got resolved. 

My questions are 
* Since devDependencies are not required at the run time, what is causing this error?
* Is adding all those babel packages to my devDependencies is the only clean solution? Since this project is not using babel, I am bit hesitant with adding it to devDependencies now.

--- a/package.json
+++ b/package.json
@@ -41,7 +41,7 @@
   "devDependencies": {
     "babel-cli": "^6.24.1",
     "babel-core": "^6.25.0",
-    "babel-eslint": "^7.2.3",
+    "babel-eslint": "^8.2.6",
     "babel-plugin-syntax-async-functions": "^6.13.0",
     "babel-plugin-syntax-flow": "^6.18.0",
     "babel-plugin-transform-builtin-extend": "^1.1.2",

--- a/package.json
+++ b/package.json
@@ -20,7 +20,9 @@
   "directories": {
     "lib": "./dist"
   },
-  "dependencies": {},
+  "dependencies": {
+    "babel-runtime": "^6.26.0"
+  },
   "peerDependencies": {
     "graphql": ">=0.9.5"
   },

